<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialogClass</class>
 <widget class="QDialog" name="EditDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="Money.qrc">
    <normaloff>:/Images/Notepad2.png</normaloff>:/Images/Notepad2.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>日期</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QDateEdit" name="dateEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::DaySection</enum>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4" rowspan="5">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buttonNew">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/edit_add.png</normaloff>:/Images/edit_add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/Floppy.png</normaloff>:/Images/Floppy.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/editdelete.png</normaloff>:/Images/editdelete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>43</width>
         <height>86</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCalculator">
       <property name="icon">
        <iconset resource="Money.qrc">
         <normaloff>:/Images/calc1.png</normaloff>:/Images/calc1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>金额</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonTax">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>加税</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxTax">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="value">
      <double>8.250000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>分类</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CategoryComboBox" name="comboBoxCategory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>用户</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="CategoryComboBox" name="comboBoxUser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>备注</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CategoryComboBox</class>
   <extends>QComboBox</extends>
   <header>CategoryComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Money.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>EditDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>EditDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>-22</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
